NOISSUE - Fix run and compile commands in README files for all services

anovakovic01 opened the pull request:
Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>

anovakovic01 committed:
Fix run command in README for websocket adapter

Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>

mijicd reviewed (commented):
Looks good. Please expand the scope of this pull request slightly and update README files of all remaining services in the same way (none of them uses Makefile).

anovakovic01 committed:
Fix compile and run commands in readme files

Fix compile and run commands in readme files, use makefile. Update
readme file for manager, http, normalizer and websocket services.

Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>

drasko commented on manager/README.md:
```diff
@@ -58,13 +58,13 @@ To start the service outside of the container, execute the following shell scrip
 
 # set the environment variables and run the service
-MF_DB_HOST=[Database host address] MF_DB_PORT=[Database host port] MF_DB_USER=[Database user] MF_DB_PASS=[Database password] MF_MANAGER_DB=[Name of the database used by the service] MF_MANAGER_PORT=[Service HTTP port] MF_MANAGER_SECRET=[String used for signing tokens] app
+MF_DB_HOST=[Database host address] MF_DB_PORT=[Database host port] MF_DB_USER=[Database user] MF_DB_PASS=[Database password] MF_MANAGER_DB=[Name of the database used by the service] MF_MANAGER_PORT=[Service HTTP port] MF_MANAGER_SECRET=[String used for signing tokens] ./build/mainflux-manager
```
I would rather suggest `make install` first. This will install the binary to `$GOBIN` (which is usually equivalent to `$GOPATH/bin`). The run it from there, or rather keep `$GOBIN` in your `$PATH` - then no prefix is needed.

anovakovic01 committed:
Add make install command to README files

Signed-off-by: Aleksandar Novakovic <anovakovic01@gmail.com>
